lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1938,7 +1944,10 @@ def _enable_ssh_on_windows(self, node: Node) -> None:
 
         log.debug("SSH port is not open, enabling ssh on Windows ...")
         # The SSH port is not opened, try to enable it.
-        public_key_data = get_public_key_data(self.runbook.admin_private_key_file)
+        (
```

Comment:
Should the Windows use the same way as Linux? Why does it need to generate key sepratedly?

Reply:
oh, I didn't realize it is windows

Reply:
removed